package.json

```diff
@@ -33,7 +33,10 @@
     "react-hook-form": "^7.51.5",
     "react-icons": "^5.2.1",
     "react-transition-group": "^4.4.5",
-    "zod": "^3.23.8"
+    "zod": "^3.23.8",
+    "embla-carousel-react": "8.2.1",
+    "embla-carousel": "8.2.1",
+    "embla-carousel-autoplay": "8.2.1"
```

Comment:
Can't we use existing antd carousel ?

Reply:
Embla carousel is better than ant carousel.

Reply:
> Embla carousel is better than ant carousel.

Can you please explain for which particular feature we should use it ?